contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -651,25 +651,22 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
         address _to,
         uint256 _tokenId
     ) internal override {
-        // Update the buyer associated with the voucher in the protocol
-        // Only when transferring, not minting or burning
-        if (_from == owner()) {
-            if (_premintStatus.committable) {
-                // Store offerId so _premintStatus can be deleted before making an external call
-                uint256 offerId = _premintStatus.offerId;
-                delete _premintStatus;
-
-                // Set the preminted token as committed
-                _committed[_tokenId] = true;
-
-                // If this is a transfer of preminted token, treat it differently
-                address protocolDiamond = IClientExternalAddresses(BeaconClientLib._beacon()).getProtocolAddress();
-                IBosonExchangeHandler(protocolDiamond).commitToPreMintedOffer(payable(_to), offerId, _tokenId);
-            } else {
-                // Already committed, treat as a normal transfer
-                onVoucherTransferred(_tokenId, payable(_to));
-            }
+        if (_premintStatus.committable) {
+            // If is commitable, invke commitToPreMintedOffer on the protocol
```

Comment:
I would move this up a line above the if statement, no ?

Also s/commitable/committable/

and 

s/invke/invoke/

Reply:
I fixed the typos.

If I move it outside the if statement, then where to move first two comments of `else if`?
They both serve the same purpose, that's why I think it makes sense to have these comments where they are.

Reply:
I guess consistency is the key, but yeah, i would have had them above, but it doesn't matter. 

If it is consistent and you are happy, then great.